README.rst

```diff
@@ -54,20 +66,18 @@ update repositories or generating patches.
 A Compiler
 -----------
 
-Since you will most likely be dealing with developers this section is probably
-not important, but just in case you get questions about compilers, here are some
-suggestions.
+If you receive questions about compilers, here are some suggestions.
 
 OS X users should be told to download XCode from the Apple App Store **ahead of
 time**. It's on the order of a couple GiB in size, so you don't want to have
 people downloading it at the sprint. After installation they should also make
 sure to install the command-line tools (e.g. in Mavericks,
 ``xcode-select --install``).
 
-If new contributors think they may be doing C development, suggest LLVM + clang
-for better error reporting than gcc.
+If new contributors think they may be doing C development, suggest the use of
+LLVM + clang as this provides better error reporting than gcc.
 
-For Windows users, tell them to download and install Visual C++ Express
+For Windows users, ask them to download and install Visual C++ Express
```

Comment:
Hmm, this is outdated (it's Visual Studio Community edition now)

I'll fix that directly after merging.